### DISCUSSION TITLE
小白求教: You exceeded your current quota, please check your plan and billing details

### DISCUSSION THREAD

**Discussion by wenjianpai** [GaiZhenbiao/ChuanhuChatGPT#397] [Q&A] [answered]
JSON解析错误。请重置对话。收到的内容: “error”: { “message”: “You exceeded your current quota, please check your plan and billing details.”, “type”: “insufficient_quota”, “param”: null, “code”: null }}

**Comment by Keldos-Li** [comment, marked as answer]
你可能把OpenAI提供的配额用完了。你可以在 https://platform.openai.com/account/usage 中查看您的用量和余额。

参见: [常见问题的该段落](https://github.com/GaiZhenbiao/ChuanhuChatGPT/wiki/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98#you-exceeded-your-current-quota-please-check-your-plan-and-billing-details)

**Reply by sc32024826** [reply]
这配额是什么情况, 我查询发现0.02 已使用 剩下的全部失效 6.1

**Reply by Keldos-Li** [reply]
> 这配额是什么情况, 我查询发现0.02 已使用 剩下的全部失效 6.1

免费额度是有使用期限的，一般为注册账号后的二至三个月，过期了就用不了了

**Reply by Shallowidea** [reply]
I just noticed that I missed the word "expired".🤣
🐂🍺熊弟
